graphql_compiler/tests/test_helpers.py

```diff
@@ -3,30 +3,56 @@
 from collections import namedtuple
 from pprint import pformat
 import re
+from typing import Any, Dict, List, Optional, Set, Tuple, Union
+from unittest import TestCase
 
 from graphql import GraphQLList, parse
-from graphql.type.definition import GraphQLInterfaceType, GraphQLObjectType
+from graphql.type.definition import (
+    GraphQLInterfaceType,
+    GraphQLList,
+    GraphQLNonNull,
+    GraphQLObjectType,
+    GraphQLScalarType,
+    GraphQLUnionType,
+)
+from graphql.type.schema import GraphQLSchema
 from graphql.utils.build_ast_schema import build_ast_schema
+from pyorient.orient import OrientDB
 import six
 import sqlalchemy
 from sqlalchemy.dialects import mssql, postgresql
 
-from graphql_compiler.schema_generation.orientdb import get_graphql_schema_from_orientdb_schema_data
-
+from ..compiler.compiler_entities import BasicBlock
 from ..compiler.subclass import compute_subclass_sets
 from ..debugging_utils import pretty_print_gremlin, pretty_print_match
-from ..macros import create_macro_registry, register_macro_edge
+from ..macros import MacroRegistry, create_macro_registry, register_macro_edge
 from ..query_formatting.graphql_formatting import pretty_print_graphql
 from ..schema import CUSTOM_SCALAR_TYPES, is_vertex_field_name
-from ..schema.schema_info import CommonSchemaInfo, DirectJoinDescriptor, make_sqlalchemy_schema_info
+from ..schema.schema_info import (
+    CommonSchemaInfo,
+    DirectJoinDescriptor,
+    SQLAlchemySchemaInfo,
+    make_sqlalchemy_schema_info,
+)
+from ..schema_generation.orientdb import get_graphql_schema_from_orientdb_schema_data
 from ..schema_generation.orientdb.schema_graph_builder import get_orientdb_schema_graph
 from ..schema_generation.orientdb.utils import (
     ORIENTDB_INDEX_RECORDS_QUERY,
     ORIENTDB_SCHEMA_RECORDS_QUERY,
 )
+from ..schema_generation.schema_graph import SchemaGraph
 from ..schema_generation.utils import amend_custom_scalar_types
+from .test_compiler import CompilerTests
+from .test_emit_output import EmitCypherTests, EmitGremlinTests, EmitMatchTests
+from .test_end_to_end import QueryFormattingTests
+from .test_ir_generation import IrGenerationTests
+from .test_macro_expansion import MacroExpansionTests
+from .test_sqlalchemy_extensions import CommonIrLoweringTests
 
 
+# The valid types that a field inside an object or interface in the GraphQL schema may be.
+GraphQLSchemaFieldType = Union[GraphQLList, GraphQLNonNull, GraphQLScalarType]
```

Comment:
I think that we probably want this type in a type defs file. I can see it being used multiple times throughout the codebase.